### PR TITLE
changing categories to enums and adding to view

### PR DESCRIPTION
**Pull request by njackson1-code**
Updated categories to enum. Also added it to view and changed the way that categories are searched for. Make sure to run migration after pulling.

### CODE REVIEW

File: app/models/item.rb [hunk 1]
```diff
@@ -11,6 +14,6 @@ class Item < ApplicationRecord
     }
     
     def self.search(query)
-        Item.all.where("lower(name) LIKE ? OR lower(category) LIKE ? OR lower(description) LIKE ?","%"+query+"%","%"+query+"%","%"+query+"%")
+        Item.all.where("lower(name) LIKE ? OR category = ? OR lower(description) LIKE ?","%"+query+"%",Item.categories[query.capitalize],"%"+query+"%")
```
Comment (Allegraknox) [inline app/models/item.rb:17]: is this sql injection safe?

Comment (amycsong98) [reply app/models/item.rb:17]: this isn't really injecting anything into the database. i think this should work.

**Review by amycsong98 — APPROVED** [review]
LGTM - did you make sure to test it locally?

**Comment by njackson1-code** [comment]
@amycsong98 

> LGTM - did you make sure to test it locally?

Yeah tested and everything seemed to be working fine

**Pull request merged**